content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -289,7 +288,7 @@ Kustomize offers composing Resources from different files and applying patches o
 #### Composing
 
 Kustomize supports composition of different resources. The `resources` field, in the `kustomization.yaml` file, defines the list of resources to include in a configuration. Set the path to a resource's configuration file in the `resources` list.
-Here is an example for an nginx application with a Deployment and a Service.
+Here is an example of an NGINX application comprised of a Deployment and a Service:
```

Comment:
Is there a reason to make nginx all caps?

Reply:
@jai , Would you change NGINX (all caps) to nginx?
A quick search in the docs comes up with nginx not NGINX.
Thanks!